zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -1217,15 +1355,10 @@ impl<F: Field> ExecutionConfig<F> {
         let final_withdraw_root_cell = self
             .end_block_gadget
             .withdraw_root_assigned
-            .borrow()
+            .lock()
```

Comment:
It seems `withdraw_root_assigned` is assigned without cocurrent (inside of a `assign_region`, not `assign_regions`) so do we really need to protect it with mutex?